velox/exec/Merge.cpp

```diff
@@ -229,29 +229,28 @@ RowVectorPtr Merge::getOutputFromSource() {
   VELOX_CHECK_NULL(spillMerger_);
   bool atEnd = false;
   output_ = sourceMerger_->getOutput(sourceBlockingFutures_, atEnd);
-  SCOPE_EXIT {
```

Comment:
Not sure what this change is about.

But, it is SCOPE_EXIT on line 179 can cause crash when exception is thrown in the function and then scope gets destroyed, calls sourceMerger_->isBlocked(), which hits the same exception again, which triggers SIGABRT.
Kevin found it.

@xiaoxmeng, @duanmeng

Reply:
similar but https://github.com/facebookincubator/velox/pull/13905 is the fix. This one fixes the spill path which is not enabled yet but only used by pyspark for now.